Rectangle/ShortcutManager.swift

```diff
@@ -81,6 +81,21 @@ class ShortcutManager {
             return
         }
         
+        // Handle double-maximize toggle for Stage Manager space
```

Comment:
Logic specific to a single window action should go into the calculation implementation for the action. In this case, maximize logic should be in the `MaximizeCalculation`.